troubleshooting/_topics/podified_logging.md

```diff
@@ -0,0 +1,3 @@
+### Podified Logging
+
+When running in podified all logs are output to the container's stdout which is picked up by Kubernetes/OpenShift and made available via `kubectl logs`/`oc logs`.
```

Comment:
TODO I'd like to add some of the examples that @jaywcarman showed where you can pass a label to `oc logs` as well as @Fryguy's json log tool

Reply:
That json tool still isn't merged: https://github.com/ManageIQ/manageiq/pull/21942

Reply:
Okay I've added an example of using the label selector, will update with the json utility once it is merged